app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -25,7 +25,7 @@ jest.mock('@/api/apiFunctions', () => ({
 }));
 
 describe('Entry Component', () => {
-  const mockUseDataStore = useDataStore as jest.Mock;
+  const mockUseDataStore = useDataStore as unknown as jest.Mock;
```

Comment:
Out of curiousity, why did you add `unknown as` here?

Reply:
> Out of curiousity, why did you add `unknown as` here?

When I'm back in my editor I'll send a screenshot explaining.

Reply:
> Out of curiousity, why did you add `unknown as` here?

@choir27, below is the error I'm getting from TypeScript without adding `as unknown`: 

Conversion of type `UseBoundStore<StoreApi<DataStore>>` to type `Mock<any, any, any>` may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to `unknown` first.

![Code - Insiders page test tsx — gridiron-survivor — Gridiron Survivor-000132](https://github.com/user-attachments/assets/08e782d8-06aa-4652-8b50-fa8e512df24f)

Reply:
Looking at this conversation from your [Loading Spinner PR](https://github.com/LetsGetTechnical/gridiron-survivor/pull/399/files/6750360ad694d646eb11d3fd45472e9977f493c5#diff-4c664975b58eb9a2b8fd8520303503caa5c63ea6e1c3f47e2682744b30c2623eR24), it seems that this shouldn't be the case. Was this conversation resolved with Shashi?

Reply:
> Looking at this conversation from your [Loading Spinner PR](https://github.com/LetsGetTechnical/gridiron-survivor/pull/399/files/6750360ad694d646eb11d3fd45472e9977f493c5#diff-4c664975b58eb9a2b8fd8520303503caa5c63ea6e1c3f47e2682744b30c2623eR24), it seems that this shouldn't be the case. Was this conversation resolved with Shashi?

@choir27 It was not resolved, which is why I've added it again here.

Reply:
☝️

Reply:
@shashilo please advise on the comments above.